docs/ramalama-ls.1.md

```diff
@@ -0,0 +1,45 @@
+% ramalama-ls 1
```

Comment:
We should just use man page aliasing.

cat podman/docs/source/markdown/links/podman-container-ls.1 
.so man1/podman-ps.1

cat ramalama-ls.1
.so man1/ramalama-ls.1

Reply:
Than add ramalama-ls to ramalama-list.md man page info

Reply:
Let's not duplicate the number of man pages, but link them together.

Everything else in this PR is great.

Reply:
Done